Piano Roll Beta

hisamov opened the pull request.

hisamov committed:
Merge branch 'master' into artur_master

hisamov committed:
Auto stash before merge of "master" and "hisamov/master"

hisamov committed:
Piano Roll Beta

hisamov committed:
Merge branch 'master' into master

hisamov committed:
fixed midiout

serifpersia commented:
The app is running again as usual. Needs some UI fixes. Few things I noticed in PianoRoll mode. Some parts of midi track slow down and speed up. The track should always be constant speed (bpm) and it should only slow down speed up based on input by user. Some midi files notes get destroyed too fast especially long notes. But for some midi files it works without problems, Maybe the midi files use sustain pedal to hold the notes. UI for pianoroll mode could change right away not when selecting midi file. We can select midi file when we get in this bigger window. I will look for more things tomorrow If I notice.

serifpersia committed:
Fixed UI, Changed back to new file dialog.

Fixed some UI bugs, and reverted back to use different file dialog. Still have to implement only .mid/.midi fliter for file selection thats missing now.

serifpersia committed:
Update UI.pde

serifpersia committed:
removed duplicate import code

serifpersia commented:
I see the color/transparency changes of the notes going down depending on velocity? I don't think we need to track velocity. Colors would be always constant light green/blue for left/right hands.

hisamov committed:
Fixes to Piano Roll

hisamov committed:
Merge remote-tracking branch 'hisamov/master'

hisamov committed:
minor fix

serifpersia committed:
Color change.

serifpersia commented:
Looks good accepting.
